packages/pds/src/api/com/atproto/admin/takeModerationAction.ts

```diff
@@ -17,7 +17,7 @@ export default function (server: Server, ctx: AppContext) {
     handler: async ({ input }) => {
       const { db, services } = ctx
       const moderationService = services.moderation(db)
-      const { action, subject, reason, createdBy } = input.body
+      const { action, subject, reason, createdBy, subjectBlobCids } = input.body
```

Comment:
Just to make sure I have it right:

Only 1 takedown can exist against a record at a time, right?

So say you have a record with `uri1` with blobs `cid1` & `cid2`.
A takedown is issued on `uri1` & `cid1`, then a second takedown is issued on `uri1`, `cid1` & `cid2`. The latter overrides the former right?

Reply:
Part of me wonders if subject cids should not be provided by the client & are just looked up in the DB when the request is made.

Like does it ever make sense to "take down post & 1 image, but leave the other"?